scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.surname, p.given_name, a.time
+FROM patients p
+INNER JOIN appointments a
+  ON a.patient_id = p.id
+INNER JOIN doctors d
+  ON a.doctor_id = d.id
+WHERE d.id = 6
```

Comment:
Since you're given the name, it would probably make the most sense to use `d.surname` for the lookup instead of `d.id`. However, this answer is not wrong.